apis/nucleus/src/components/Cell.jsx

```diff
@@ -470,6 +470,9 @@ const Cell = forwardRef(
         getQae() {
           return state.sn.generator.qae;
         },
+        getExtDefinition() {
+          return state.sn.generator.definition.ext;
```

Comment:
When the chart is in view data the definition I guess coming from the table and therefor viewData is false.

Reply:
Somehow when I have boxplot the `ext` is undefined meanwhile in sense it has data.